extensions/register.go

```diff
@@ -4,4 +4,7 @@ package extensions
 // this directory has 0 dependencies, so it can import anything
 // it is imported by cmd/kwild/main.go, so any other files in this directory will be compiled
 
-import _ "github.com/kwilteam/kwil-db/extensions/oracles/eth_deposits"
+import (
+	_ "github.com/kwilteam/kwil-db/extensions/oracles/eth_deposits"
+	_ "github.com/kwilteam/kwil-db/extensions/oracles/spam"
```

Comment:
Did you need to `git add` a new `spam` package for testing?

Reply:
oops. I actually meant to remove the spam package. I think we need a more formal way of testing spam, but for now I was just testing manually with a basic for loop

Reply:
While you were gone, we discussed a testing framework for compiled extensions. This would apply to oracles as well. I don't really want to implement this before the release